interfaces/builtin/system_packages_doc.go

```diff
@@ -81,6 +81,30 @@ func (iface *systemPackagesDocInterface) AppArmorConnectedPlug(spec *apparmor.Sp
 	apparmor.GenWritableProfile(emit, "/usr/share/cups/", 3)
 	apparmor.GenWritableProfile(emit, "/usr/share/gimp/2.0/", 3)
 	apparmor.GenWritableProfile(emit, "/usr/share/libreoffice/", 3)
+
+	if plug.Snap().Base == "bare" {
+		// The bare snap does not have enough mount points, causing us to create a mimic over /
+		// which only works when snap-update-ns is invoked without the sandbox by snapd. When invoked
+		// from starting snap via the snap-rup -> snap-confine -> snap-update-ns chain, the permissions
+		// are not sufficient.
+		//
+		// In essence, constructing this sort of mimic requires nearly arbitrary writes/mounts at root:
+		// See bug comments for details LP:#2044335
+		emit(`
+  # Writable mimic over / - extra permissions generalized
+  "/**" rw,
+  "/tmp/.snap/usr/**" rw,
```

Comment:
The template for snap-update-ns already has `/tmp/.snap/{,**} rw` which should be a super-set of `"/tmp/.snap/usr/**" rw,` and so I don't think this should be needed.

Reply:
On second thought, given we now also have `/**` directly above it, should this `/tmp/.snap/user/**` rule be needed at all?

Reply:
I agree, it can be reduced.

Reply:
Done